composer.json

```diff
@@ -17,7 +17,8 @@
     ],
     "require": {
         "php": "^7.2",
-        "illuminate/support": "5.6.*|5.7.*|5.8.*|^6.0"
+        "illuminate/support": "5.6.*|5.7.*|5.8.*|^6.0",
+        "symfony/process": "^5.0"
```

Comment:
Should this be `"symfony/process": "^4.0|^5.0"`?

Reply:
Yes, you are right.